Assets/Scripts/Projectiles/BulletEffect.cs

```diff
@@ -0,0 +1,11 @@
+namespace Projectiles
+{
+    public enum BulletEffect
+    {
+        NONE,
+        SLOW,
+        POISON,
+        AOE,
+        PIERCE,
```

Comment:
Name 'NONE' does not match rule 'Enum members'. Suggested name is 'None'.

Nu suntem în Java land. Tu nu scriai în Rider? Rider ar trebui să-ți indice majoritatea chestiilor de care tot comentez eu.